include/TPP/TransformUtils.h

```diff
@@ -78,6 +78,12 @@ bool isMatmulOp(Operation *op,
 bool hasMulAddBody(linalg::LinalgOp linalgOp,
                    SmallVectorImpl<Value> *capturedOperands = nullptr);
 
+// Rewrite scf.for to scf.forall. Assumes the loop to be parallel and
+// marked with `kLoopId`.
+constexpr const static llvm::StringLiteral kLoopId = "parallel";
```

Comment:
I'd name this `kLoopParallel` to make it clear throughout the code without having to look at the declaration.